web/src/user_groups.ts

```diff
@@ -291,7 +292,7 @@ export function get_realm_user_groups_for_dropdown_list_widget(
             };
         });
 
-    if (require_system_group) {
+    if (!page_params.development_environment || require_system_group) {
```

Comment:
We can also pass the server level setting in `page_params` response, but since we do not pass it alraedy, I have used `development_environment` check for now. We can remove this once we allow user defined and anonymous groups in production.

Reply:
This seems OK for now, thanks!